File: src/main/java/net/shadowfacts/craftingslabs/items/ModItems.java
package net.shadowfacts.craftingslabs.items;

import net.minecraftforge.fml.common.registry.GameRegistry;
import net.shadowfacts.craftingslabs.CraftingSlabs;

/**
 * @author shadowfacts
 */
public class ModItems {

	public ItemCraftingSlab craftingSlab;
	public ItemFurnaceSlab furnaceSlab;

	public void register() {
		craftingSlab = GameRegistry.register(new ItemCraftingSlab(CraftingSlabs.blocks.craftingSlab));
		furnaceSlab = GameRegistry.register(new ItemFurnaceSlab(CraftingSlabs.blocks.furnaceSlab));
	}

}

File: src/main/kotlin/net/shadowfacts/craftingslabs/item/ModItems.kt
package net.shadowfacts.craftingslabs.item

import net.shadowfacts.shadowmc.item.ModItems

/**
 * @author shadowfacts
 */
object ModItems : ModItems() {

	val craftingSlab = ItemCraftingSlab()
	val furnaceSlab = ItemFurnaceSlab()

	override fun init() {
		register(craftingSlab)
		register(furnaceSlab)
	}

}